File: Question9/Question9.cs
using System.CodeDom.Compiler;
using System.Collections.Generic;
using System.Collections;
using System.ComponentModel;
using System.Diagnostics.CodeAnalysis;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.Serialization;
using System.Text.RegularExpressions;
using System.Text;
using System;

class Solution {

    // Complete the bankersAndClients function below.
    static int bankersAndClients(int c,int b,int[][] container) {
        return -1;

    }

    static void Main(string[] args) {

		int[] ar = new int[2];
		ar = Array.ConvertAll(Console.ReadLine().Split(' '), containerTemp => Convert.ToInt32(containerTemp));
		int c = ar[0];
		int b = ar[1];
		int[][] container = new int[b][];

		for (int i = 0; i < b; i++) {
			container[i] = Array.ConvertAll(Console.ReadLine().Split(' '), containerTemp => Convert.ToInt32(containerTemp));
		}

		int result = bankersAndClients(c,b,container);
            
        // Do not remove below line
		Console.WriteLine(result);
        // Do not print anything after this line

    }
}

File: Question9/Question9.js
'use strict';


process.stdin.resume();
process.stdin.setEncoding('utf-8');

let inputString = '';
let currentLine = 0;

process.stdin.on('data', inputStdin => {
    inputString += inputStdin;
});

process.stdin.on('end', _ => {
    inputString = inputString.replace(/\s*$/, '')
        .split('\n')
        .map(str => str.replace(/\s*$/, ''));

    main();
});

function readLine() {
    return inputString[currentLine++];
}

function bankersAndClients(c, b, time) {

    // Complete the bankersAndClients function below.
    return -1;
}

function main() {
    var ar = readLine().split(' ').map(Number);
    var c = ar[0];
    var b = ar[1];
    var time = [];
    for (var i = 0; i < b; i++) {
        time[i] = readLine().split(' ');
        time[i] = time[i].map(Number);
    }

    var result = bankersAndClients(c, b, time);
    // Do not remove below line
    console.log(result);
    // Do not print anything after this line
}